src/Helper/Fields/Field_Repeater.php

```diff
@@ -107,6 +107,22 @@ public function get_repeater_form_data( $data, $value, $field ) {
 				if ( isset( $form_data['field'] ) ) {
 					$data = array_replace_recursive( $data, [ $id => $form_data['field'] ] );
 				}
+
+				/* Allow HTML to be included in $form_data */
+				if ( isset( $form_data['html_id'] ) ) {
+					$field_id = 0;
+					$content  = '';
+					/* Extract field id and html content */
+					foreach ( $form_data['html_id'] as $key => $value ) {
+						$field_id = $key;
```

Comment:
Won't this loop through the entire array and only set the last array item's `$key` and `$value` as `$field_id` and `$content` respectively? 

It feels like you should rename `$key` to `$field_id` and `$value` to `$content`, and then remove lines 121-123 inside this array. You'd then be able to remove lines 113-114.